hoomd/mesh.py

```diff
@@ -128,7 +123,7 @@ def bonds(self):
 
     @property
     def size(self):
-        """(`uint32``): Number of triangles in the mesh."""
+        """(``uint32``): Number of triangles in the mesh."""
```

Comment:
```suggestion
        """(int): Number of triangles in the mesh."""
```
Unless this is a NumPy type the type is just int in Python.